examples/docker-agent/agent.py

```diff
@@ -1,375 +0,0 @@
-"""Implementation of a simple deterministic agent using Docker."""
-
-from collections import defaultdict
-import random
-
-import numpy as np
-
-from common import DockerAgent
-from pommerman.envs import utility
-
-
-class Agent(DockerAgent):
-    def __init__(self):
-        self._agent = PommermanTestAgent()
-
-    def act(self, observation, action_space):
-        return self._agent.act(observation, action_space)
-
-
-class PommermanTestAgent(object):
```

Comment:
Nice! Thanks for culling this. I had been meaning to refactor this away given that it's duplicated.